src/main/java/team/chisel/item/chisel/ChiselController.java

```diff
@@ -44,6 +44,9 @@ public void onInteract(PlayerInteractEvent event) {
 
 		switch (event.action) {
 		case LEFT_CLICK_BLOCK:
+			boolean shifting = false;
```

Comment:
Yes but I made it so in case you need to use it somewhere else later on you have it